docs/expression_syntax.md

```diff
@@ -6,11 +6,13 @@ In order of evaluation priority. _(top to bottom, left to right)_
 | 5     | `()`                 | Priority Expression                                   | left-to-right |
 | 4     | `not`, `+`, `-`      | Unary Logical NOT<br>Unary Positive<br>Unary negative | right-to-left |
 | 3     | `<`, `<=`, `>`, `>=` | Relational Operators                                  | left-to-right |
+| 3     | `*`, `/`             | Multiple and Divison Operators                        | left-to-right |
 | 2     | `==`, `!=`           | Equality Operators                                    | left-to-right |
+| 2     | `+`, `-`             | Addition and Subtraction Operators                    | left-to-right |
```

Comment:
Should we place these operations on separate levels from the the relational and equality operators? I believe we need to evaluate these operators before we can compare relataional or equality operators.

At the moment according to this documentation the following statement would resolve to false with the current priority order: `2 + 4 > 5`

Please confirm the priority order for these new operators and the behavior.

Reply:
These changes do not support expressions like `2 + 4 > 5` for now. It is a relatively simple change from here but did not include it

Reply:
While we don't support that expression type right now the documentation is miss leading and violates the standard order of operations. We need to correct this.